charts/ingress-nginx/tests/admission-webhooks/job-patch/job-createSecret_test.yaml

```diff
@@ -76,3 +76,46 @@ tests:
                           fieldRef:
                             apiVersion: v1
                             fieldPath: metadata.namespace
+
+  - it: should create a Job with resizePolicy if `controller.admissionWebhooks.createSecretJob.resizePolicy` is set and Kubernetes version >= 1.30
+    capabilities:
+      majorVersion: 1
+      minorVersion: 30
+    set:
+      controller.admissionWebhooks.createSecretJob.resizePolicy:
+        - resourceName: cpu
+          restartPolicy: NotRequired
+        - resourceName: memory
+          restartPolicy: RestartContainer
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[0].resizePolicy[0].resourceName
+          value: cpu
+      - equal:
+          path: spec.template.spec.containers[0].resizePolicy[0].restartPolicy
+          value: NotRequired
+      - equal:
+          path: spec.template.spec.containers[0].resizePolicy[1].resourceName
+          value: memory
+      - equal:
+          path: spec.template.spec.containers[0].resizePolicy[1].restartPolicy
+          value: RestartContainer
```

Comment:
`value` can also be a whole array. You do not need to check each element of the array.